pkg/berglas/create.go

```diff
@@ -63,6 +63,10 @@ type SecretManagerCreateRequest struct {
 
 	// Plaintext is the plaintext to store.
 	Plaintext []byte
+
+	// Locations is an array indicating the canonical IDs (e.g. "us-east1") of the locations to the replicate data at.
+	// This defaults to the automatic replication policy when not specified. An empty array is not allowed.
+	Locations []string
```

Comment:
Add a note that this only applies when the storage is Secret Manager and line wrap please.

Reply:
I'd be more than willing to add the note if you'd like. But isn't this struct specific for Secret Manager already?

Reply:
Yea, sorry, you're right.